zeppelin-zengine/pom.xml

```diff
@@ -257,6 +258,12 @@
       <version>1.5</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.json</groupId>
+      <artifactId>json</artifactId>
+      <version>${org.json.version}</version>
```

Comment:
is this the code/license?

https://github.com/stleary/JSON-java/blob/master/LICENSE

sounds like we need to include in the LICENSE file

Reply:
Thanks, I'll add that.